install/install.go

```diff
@@ -141,23 +141,41 @@ func checkPods(c *k8s.Client, o Options) {
 			break
 		}
 	}
-	probeData, err := probe.ProbeRunningKubeArmorNodes(c, probe.Options{
-		Namespace: o.Namespace,
-	})
-	if err != nil || len(probeData) == 0 {
-		return
-	}
-	enforcing := true
-	for _, k := range probeData {
-		if k.ActiveLSM == "" || !k.ContainerSecurity {
-			enforcing = false
-			break
+	fmt.Print("\n🔧  Verifying KubeArmor functionality (this may take upto a minute) ...")
+	ctx, cancel := context.WithTimeout(context.Background(), 40*time.Second)
+	defer cancel()
+
+	for {
+		select {
+		case <-time.After(1 * time.Second):
```

Comment:
It does not actually run after 1s, the process of probeRead in itself takes 5-6 second. I have seen anywhere between 2-4 attempts before we get the data. So set the total timeout to 40 seconds.

Reply:
Barun, on a machine with a slower internet connection because I've seen even after 4 steps, the output is missing the infos? Should we increase this to 10sec or more? 
Also, it fixes https://github.com/kubearmor/kubearmor-client/issues/297

Reply:
Yes, I tested the PR. I think the existing method was good too, in which no delay was there after all the services were running. By using this, we are making an assumption that the user will invoke `karmor probe` immediately after installing KubeArmor which may not be the case. Open to more thoughts on this.

Reply:
> By using this, we are making an assumption that the user will invoke `karmor probe` immediately after installing KubeArmor which may not be the case. 

Not sure what you mean by making an assumption, we want users to know whether on installation KubeArmor enforcement works or not. Ref #334

Reply:
It works good. I missed the context. Thanks for the ref.

Reply:
Anurag, do you think we need to increase the time duration still?

Reply:
No, I think this is more than enough. Initially I thought we were doing this so that we can get some time till the data gets written to `/tmp/KarmorProbe.cfg` which was my misunderstanding. I was thinking more in lines of #297